packages/tailwindcss-language-server/src/oxide.ts

```diff
@@ -58,10 +58,32 @@ declare namespace OxideV3 {
   }
 }
 
+// This covers the Oxide API from v4.1.0+
+declare namespace OxideV5 {
+  interface GlobEntry {
+    base: string
+    pattern: string
+    negated: boolean
+  }
```

Comment:
Unless you want to keep using `GlobEntry` and `SourceEntry` as 1 shared thing?
```suggestion
  }
  
  interface SourceEntry {
    base: string
    pattern: string
    negated: boolean
  }
```